src/main/java/org/opentripplanner/middleware/tripmonitor/jobs/CheckMonitoredTrip.java

```diff
@@ -421,23 +424,30 @@ private void sendNotifications() {
         // Update push notification devices count, which may change asynchronously
         NotificationUtils.updatePushDevices(otpUser);
 
-        if (notifications.size() == 0) {
+        boolean hasInitialReminder = initialReminderNotification != null;
+
+        if (notifications.size() == 0 && !hasInitialReminder) {
```

Comment:
Could replace with `notifications.isEmpty()`

Reply:
Fixed in https://github.com/ibi-group/otp-middleware/pull/205/commits/b5d2fce5f360398be8caabd60ffa7a4bff102068.